Frontend/ui-customer/src/Flow.purs

```diff
@@ -378,6 +396,7 @@ chooseLanguageScreenFlow = do
   case flow of
     NextScreen language -> do
                             setValueToLocalStore LANGUAGE_KEY language
+                            void $ pure $ setCleverTapUserProp "Preferred Language" language
```

Comment:
@ashkritidewan2000  language is being changed in selectLanguageScreenFlow as well. Please call this function there as well.

Reply:
Done